build-system/babel-plugins/babel-plugin-transform-rename-privates/index.js

```diff
@@ -28,15 +28,16 @@ module.exports = function (babel) {
         return;
       }
 
-      const {name} = key.node;
+      const {node} = key;
+      const {name} = node;
       if (name.endsWith('_AMP_PRIVATE_')) {
         return;
       }
 
-      if (!name.endsWith('_')) {
+      if (!name.endsWith('_') || name === '__proto__') {
         return;
       }
-      key.replaceWith(t.identifier(`${name}AMP_PRIVATE_`));
+      key.replaceWith(t.inherits(t.identifier(`${name}AMP_PRIVATE_`), node));
```

Comment:
When we replaced the node, it forgets what the original name is. We have to use `t.inherits` to pass the data from the original Identifier to the new Identifier.